main.js

```diff
@@ -0,0 +1,23 @@
+function daysInYear(year) {
+  try {
+    if (!Number.isInteger(year)) {
+      throw new SyntaxError('exeption');
+    }
+    const daysInFebruary = new Date(year, 1, 29).getDate();
+    if (daysInFebruary === 29) {
+      console.log(366);
```

Comment:
This method should return number of days. Currently it prints it into the console, which is a different thing then returning value.